modules/openthread/openthread.c

```diff
@@ -182,15 +183,15 @@ static bool ot_setup_default_configuration(void)
 		return false;
 	}
 
-	net_bytes_from_str(xpanid.m8, 8, (char *)OT_XPANID);
+	bytes_from_str(xpanid.m8, 8, (char *)OT_XPANID);
 	error = otThreadSetExtendedPanId(openthread_instance, &xpanid);
 	if (error != OT_ERROR_NONE) {
 		LOG_ERR("Failed to set %s [%d]", "ext PAN ID", error);
 		return false;
 	}
 
 	if (strlen(OT_NETWORKKEY)) {
-		net_bytes_from_str(networkKey.m8, OT_NETWORK_KEY_SIZE, (char *)OT_NETWORKKEY);
+		bytes_from_str(networkKey.m8, OT_NETWORK_KEY_SIZE, (char *)OT_NETWORKKEY);
```

Comment:
Probably we don't need it, because otThread methods will handle it properly anyway due to internal checks, but please note that below we are passing buffer that may contain trash, as output result was not verified.

Reply:
Ok, I've added a check here.